src/components/pages/PartnersAdmin/PartnersAdmin.js

```diff
@@ -1,121 +1,313 @@
+    const { isEditing, partnerEditId } = this.state;
+    if (isEditing) {
+      partnerRequests
         .editPartner(partnerEditId, newPartner)
         .then(() => {
-                this.getPartners();
-                this.setState({
-                    showModal: false,
-                    isEditingPartner: false,
-                    partnerEditId: '-1',
-                });
-            })
-            .catch(error => console.error('There Was An Error Editing Your Partner', error));
-        } else {
-        partnerRequests
+          this.getPartners();
+          this.setState({
+            showModal: false,
+            isEditing: false,
+            partnerEditId: '-1',
+          });
+        })
+        .catch(error => console.error('There Was An Error Editing Your Parting Pets Partner', error));
+    } else {
+      partnerRequests
         .createPartner(newPartner)
-        .then(() => {
-        this.getPartners();
-        this.setState({ showModal: false });
+        .then((result) => {
+          this.getPartners();
+          this.setState({ showModal: false });
         })
-        .catch(error => console.error('There was an error creating the partner'));
+        .catch(error => console.error('There Was An Error Creating Your New Parting Pets Partner'));
     }
-};
-
-passPartnerToEdit = partnerId => {
-    partnerRequests.getSinglePartner(partnerId)
-    .then((result) => {
-        this.setState({ 
-            isEditingPartner: true, 
-            partnerEditId: partnerId,
-            partnerToEdit: result 
-        });
-            this.showModal();
-    }); 
-}
+  };
 
-  deleteOnePartner = (partnerId) => {
+  // Delete Partner Request //
+  deletePartner = (partnerId) => {
     partnerRequests
       .deletePartner(partnerId)
       .then(() => {
-        partnerRequests.getAllPartners().then((partners) => {
-          this.setState({ partners });
+        this.getPartners();
+      })
+      .catch(error => console.error('error with deleting this parting pets partner.', error));
+  };
+
+  // Edit Partner Request //
+  editPartner = (partnerId) => {
+    partnerRequests
+      .getSinglePartner(partnerId)
+      .then((partner) => {
+        this.setState({
+          isEditing: true,
+          partnerEditId: partnerId,
+          partnerToEdit: partner,
         });
+        this.showModal();
       })
-      .catch((err) => {
-        console.error('error in deleting the partner', err);
-      });
+      .catch(error => console.error('There Was An Issue Getting Your Parting Pets Partner To Update', error));
   };
 
-render() {
-    const { partners, isEditingPartner, partnerToEdit } = this.state;
-    const partnersComponents = partners.map(partner => (
-     <PartnerItems
+  render() {
+    const { userObject } = this.props;
+    const { partners, isEditing, partnerToEdit } = this.state;
+
+    const printPartner = partners.map((partner, index) => (
+      <ManagePartnersTable
```

Comment:
Getting an error in the console about the `partner` proptype should be a function but got an object

```Warning: Failed prop type: ManagePartnersTable: prop type `partner` is invalid; it must be a function, usually from the `prop-types` package, but received `object`.
    in ManagePartnersTable (at PartnersAdmin.js:253)```